extensions/Cargo.lock

```diff
@@ -522,7 +522,7 @@ dependencies = [
 [[package]]
 name = "gamescope-x11-client"
 version = "0.1.0"
-source = "git+https://github.com/ShadowBlip/gamescope-x11-client?branch=main#3a0cbe64ba60dffb5ad85f156101c056f764e659"
+source = "git+https://github.com/ShadowBlip/gamescope-x11-client?branch=main#deeab5be067bfbb2add2446d8c7fbcbeba7c8c7f"
```

Comment:
Why did you commit this?

Reply:
To update the version of the `gamescope-x11-client` crate.

Reply:
Isn't that done in the toml and the lock gets updated and committed during CI?